### DISCUSSION TITLE
Burn subtitles in middle of screen

### DISCUSSION THREAD

**Discussion by hartent910** [SubtitleEdit/subtitleedit#6136] [Ideas]
I would like the burned subtitles to display in the center of the entire video (not just the subtitle box at bottom). I see I can highlight-all in the text and change alignment but this only changes in the subtitle box. 

see https://www.youtube.com/watch?v=c72aqAYqVEo for an example.

Also make the font bigger as well. 

THanks for this awesome software!

**Comment by niksedk** [comment]
Try to switch to subtitle format `Advanced Sub Station Alpha` - change alignment + font size in "ASSA styles" (the "A" icon in the toolbar)

**Comment by hartent910** [comment]
That did it! Thank you so much for making this software. It does better than Kapwing or another crap software that I have. The translations are better and the interface is easier to view and use. Also, it is super that the videos render in 4K (Kapwing doesn't do that and they charge a monthly fee.)

One thing that I will note for others who use ASSA styles feature, the fact that you are changing the location of the subtitles will not appear on the working screen where the subs are displayed; but will show as you set them when the video is rendered.  That goes for other styles you may change like having an opaque box with the subtitles.   I suggest you do a short video to test the settings first and then make the actual video with the same settings. 

Again, thanks so much for this amazing tool!

**Reply by niksedk** [reply]
What video player are you using inside SE? Mpv should support positioning in the SE video player preview.

**Comment by hartent910** [comment]
mpv. Also note I am using portable version. I did notice if I close the app and reload, then it does show in the new position.
